src/styles/header.scss

```diff
@@ -0,0 +1,71 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0 auto;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  transition: all 300ms;
+
+  &__logo-img {
+    height: $logo-img-size;
+    width: $logo-img-size;
+    display: block;
+  }
+
+  &__link {
+    text-decoration: none;
+    line-height: 0;
+  }
+}
+
+.nav {
+  &__list {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    align-content: center;
+    margin: 0;
+    padding-inline-start: 0;
+  }
+
+  &__item {
+    position: relative;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $nav-link-color;
+    height: 60px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    text-transform: uppercase;
+    padding-bottom: 0;
+    transition: all 300ms;
+
+    &:hover,
+    &:active {
+      color: $main-color;
+    }
+  }
+}
+
+.is-active {
+  color: $main-color;
+
+  &::after {
+    content: '';
+    width: 100%;
+    height: 4px;
+    background-color: $main-color;
```

Comment:
Ensure that the variable `$main-color` is defined in your SCSS files. If it's not defined, it will cause an error during compilation.